Jenkinsfile

```diff
@@ -250,15 +272,15 @@ pipeline {
           steps {
             sh 'export'
             sh 'make distclean; rm -rf .ros; rm -rf .gazebo'
-            sh 'git fetch --tags'
-            sh 'make posix_sitl_default'
-            sh 'make posix_sitl_default sitl_gazebo'
-            sh './test/rostest_px4_run.sh mavros_posix_test_mission.test mission:=vtol_new_1 vehicle:=standard_vtol'
-            sh './Tools/ecl_ekf/process_logdata_ekf.py `find . -name *.ulg -print -quit`'
+            unstash 'px4_sitl_package'
+            sh 'tar -xjpf build/posix_sitl_default/px4-posix_sitl_default*.bz2'
+            sh 'mv px4-posix_sitl_default*/bin/px4 px4-posix_sitl_default*/share/px4/'
```

Comment:
Why do we need the move?

Reply:
I just found it a bit weird that the package contents need to be massaged to work.

Reply:
Agree. It is weird. To avoid this we can either 1) change cpack target destination for px4 -- don't use bin and share dirs or 2) don't launch px4 in roslaunch anymore -- script the launch instead

Reply:
I would do whatever's easier and simpler at this point. Keep in mind CI is effectively the only consumer of the package at the moment, so it might as well be made to fit.
I don't have an opinion about dropping roslaunch yet. What's better long term?

Reply:
At this point, where the pkg is only for CI, I'd rather change the CPack destination dirs. Option 2 is a more invasive change (new launch file and script, more testing, and so on). Long term, if users are to `sudo dpkg -i px4.deb`, this would be the more appropriate approach.